permuta/cli.py

```diff
@@ -46,58 +37,81 @@ def has_regular_insertion_encoding(args):
         print(f"{perm_class} does not have a regular insertion encoding")
 
 
-def get_lex_min(args):
-    basis = parse_basis(args)
-    print("_".join(str(p) for p in lex_min(basis)))
-
-
-basis_str = (
-    "The basis as a string where the permutations are separated by '_'"
-    " (e.g. '231_4321')"
-)
-
-parser = argparse.ArgumentParser(description="A set of tools to work with permutations")
-subparsers = parser.add_subparsers(title="subcommands")
-
-
-# The count command
-count_parser = subparsers.add_parser(
-    "count", description="A tool to quickly get the enumeration of permutation classes"
-)
-count_parser.set_defaults(func=enumerate_class)
-count_parser.add_argument(
-    "basis", help=basis_str,
-)
-
-# The insenc command
-insenc_parser = subparsers.add_parser(
-    "insenc",
-    description="A tool to check if a permutation class has a regular insertion"
-    " encoding.",
-)
-insenc_parser.set_defaults(func=has_regular_insertion_encoding)
-insenc_parser.add_argument(
-    "basis", help=basis_str,
-)
-
-# The lexmin command
-lexmin_parser = subparsers.add_parser(
-    "lexmin",
-    description="A tool that returns the 0-based lexicographically minimal "
-    "representation of the basis.",
-)
-lexmin_parser.set_defaults(func=get_lex_min)
-lexmin_parser.add_argument(
-    "basis", help=basis_str,
-)
-
-
-def main():
+def get_lex_min(args: argparse.Namespace) -> None:
+    """Prints the 0-based lexicographically minimal representation of the basis."""
+    basis = Basis.from_string(args.basis)
+    print(f'{{{",".join(str(perm) for perm in lex_min(basis))}}}')
```

Comment:
Can you change this to '_' as thats what we've used in the past, and it will be better for searching old logs.